apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/AspNetCoreSnippets.cs

```diff
@@ -45,5 +48,36 @@ public void LogMessage(ILoggerFactory loggerFactory)
             logger.LogInformation("This is a log message.");
         }
         // End sample
+
+        
+        private class Trace
+        {
+            // Sample: RegisterGoogleTracer
+            public void ConfigureServices(IServiceCollection services)
+            {
+                string projectId = "[Google Cloud Platform project ID]";
+                services.AddGoogleTrace(projectId);
+            }
+
+            public void Configure(IApplicationBuilder app)
+            {
+                // Use at the start of the request pipeline to ensure the entire
+                // request is traced.
+                app.UseGoogleTrace();
+            }
+            // End sample
+
+            // Sample: UseTracer
+            /// <param name="tracer">Populated by dependency injection.</param>
```

Comment:
(Hmm, could have sworn I added this comment the first time around. Not sure where it went) I would either remove this triple-slash (and use a normal comment inside the method perhaps) or add a \<summary\> description.